manifests/latest/vsphere-7.0u1/vanilla/deploy/validatingwebhook.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sc-validation-webhook-svc
+  namespace: kube-system
+  labels:
+    app: vsphere-csi-controller
+spec:
+  ports:
+    - port: 443
+      targetPort: 8443
+  selector:
+    app: vsphere-csi-controller
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validation.csi.vsphere.vmware.com
+webhooks:
+  - name: scvalidation.csi.vsphere.vmware.com
+    clientConfig:
+      service:
+        name: sc-validation-webhook-svc
```

Comment:
remove sc from here. Let's have a single validation webhook for everything.

Reply:
service name is now set to `validation-webhook-svc`